code/modules/clothing/outfits/plasmaman.dm

```diff
@@ -16,6 +16,12 @@
 	head = /obj/item/clothing/head/foilhat/plasmaman
 	uniform = /obj/item/clothing/under/plasmaman
 
+/datum/outfit/plasmaman/radio
+	name = "Radiohost Plasmaman"
+
+	head = /obj/item/clothing/head/helmet/space/plasmaman/hat
```

Comment:
Iirc you're using the magician's set here. I'd at least change the hat, to a normal service envirohelm